packages/commonwealth/client/scripts/controllers/server/search.ts

```diff
@@ -89,17 +89,45 @@ class SearchController {
       }
 
       if (scope.includes(SearchScope.Members)) {
-        const addrs = await this.searchMentionableAddresses(
+        const profiles = await this.searchMentionableProfiles(
           searchTerm,
-          { pageSize, chainScope },
-          ['created_at', 'DESC']
+          chainScope
         );
 
-        searchCache.results[SearchScope.Members] = addrs
-          .map((addr) => {
-            addr.SearchContentType = SearchContentType.Member;
-            addr.searchType = SearchScope.Members;
-            return addr;
+        searchCache.results[SearchScope.Members] = profiles
+          .map((profile) => {
+            const profileAsMember: any = {
+              id: profile.id,
+              profile_id: profile.id,
+              address: profile.addresses[0]?.address,
```

Comment:
Any reason we take the first address/chain combo?

Reply:
A profile may have multiple associated addresses, as far as I can tell. But everywhere we render "members", it only shows a single address.

Here's an example of the new payload:
```
{
    "id": 94306,
    "user_id": 97027,
    "profile_name": "rbcw",
    "addresses": [
        {
            "id": 110358,
            "chain": "ethereum",
            "address": "0xd56BBB2072085e38705f29349F2E89f079c93659"
        },
        {
            "id": 110398,
            "chain": "dydx",
            "address": "0xd56BBB2072085e38705f29349F2E89f079c93659"
        }
    ]
}
```

If the `searchProfiles` API request is scoped by chain, then there will only be a single address populated. If the search is unscoped, then there may be multiple addresses for a profile.

Once a user clicks on a member, it's going to navigate by profile ID, so it doesn't actually matter which address/chain is used.

Reply:
If my understanding of profiles/addresses is correct, we may need to redesign some parts of the app?